src/test/cpp/util/transformer.cpp

```diff
@@ -116,14 +116,25 @@ void Transformer::createSedCommandFile(const std::string& regexName,
 
 	std::string tmp;
 
+	auto sedsaniziter = [] (const std::string &in, const std::string &sedseperator = "Q") {
```

Comment:
`sedsaniziter` contains a spelling error most likely, prefer `sedSanitizer`. `sedseperator` should follow that naming with `sedSeperator` as well. We mostly follow Java naming conventions.

Please change the references to be `const std::string& ...`, because most other code does so already.

Reply:
should be fixed now by 38729af.